node/src/actors/json_rpc/json_rpc_methods.rs

```diff
@@ -77,6 +77,9 @@ pub fn jsonrpc_io_handler(
     io.add_method("knownPeers", |_params: Params| known_peers());
     io.add_method("nodeStats", |_params: Params| node_stats());
     io.add_method("getMempool", |params: Params| get_mempool(params.parse()));
+    io.add_method("getConsensusConstants", |params: Params| {
+        get_consensus_constants(params.parse())
```

Comment:
Should this method be marked as sensitive?

Reply:
why do you think that could be marked as sensitive?

Reply:
It allows you to know to which network is the node connected to (testnet or mainnet), but I'm not sure if that is sensitive or not.

Reply:
I think it is not sensitive. It is suppose that most of the nodes will use the same consensus constants

Reply:
True, I thought it can be sensitive if we add other config values to the output, but that will be a different `getConfig` method, not this one. So let's leave this as not sensitive.